src/Umbraco.Cms.Integrations.DAM.Aprimo/App_Plugins/UmbracoCms.Integrations/DAM/Aprimo/Render/AprimoAsset.cshtml

```diff
@@ -1,11 +1,53 @@
 ﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Umbraco.Cms.Integrations.DAM.Aprimo.Models.ViewModels.AprimoAssetViewModel>
 
-<link href="~/App_Plugins/UmbracoCms.Integrations/DAM/Aprimo/css/aprimo.css" rel="stylesheet"/>
+<style type="text/css">
```

Comment:
As per other comment, I'm wondering if we should provide this.  Of course it makes it look better to have some styling, but I'm wondering what happens if someone wants to update it.  They can't override this in a stylesheet, and they can't edit the file.

So maybe we provide in the package just the rendering view - with CSS classes.

And provide in documentation a sample set of styling that makes the rendering look reasonable.

The implementer can then either use our partial view and apply styles.  Or, if that's not enough, create their own rendering component using ours as a reference.